cg/services/orders/submitters/order_submitter_registry.py

```diff
@@ -76,97 +73,97 @@ def get_order_submitter(self, order_type: OrderType) -> OrderSubmitter:
 order_service_mapping = {
     OrderType.BALSAMIC: (
         OrderLimsService,
-        ValidateCaseOrderService,
+        BalsamicValidationService,
         StoreCaseOrderService,
         CaseOrderSubmitter,
     ),
     OrderType.BALSAMIC_QC: (
         OrderLimsService,
-        ValidateCaseOrderService,
+        BalsamicValidationService,
         StoreCaseOrderService,
         CaseOrderSubmitter,
     ),
     OrderType.BALSAMIC_UMI: (
         OrderLimsService,
-        ValidateCaseOrderService,
+        BalsamicValidationService,
         StoreCaseOrderService,
         CaseOrderSubmitter,
     ),
     OrderType.FASTQ: (
         OrderLimsService,
-        ValidateFastqOrderService,
+        BalsamicValidationService,
```

Comment:
I guess these are just placeholders for now?

Reply:
Yes, let's not forget to put in the correct ones 😅